cmd/helm/status.go

```diff
@@ -2,16 +2,28 @@ package main
 
 import (
 	"github.com/deislabs/porter-helm/pkg/helm"
+	"github.com/deislabs/porter/pkg/printer"
 	"github.com/spf13/cobra"
 )
 
 func buildStatusCommand(m *helm.Mixin) *cobra.Command {
+	opts := struct {
+		rawFormat string
+		format    printer.Format
+	}{}
 	cmd := &cobra.Command{
 		Use:   "status",
-		Short: "Execute the status functionality of this mixin",
+		Short: "Print the status of the helm components in the bundle",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+			opts.format, err = printer.ParseFormat(opts.rawFormat)
+			return err
+		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return m.Status()
+			return m.Status(printer.PrintOptions{Format: opts.format})
 		},
 	}
+
+	cmd.Flags().StringVarP(&opts.rawFormat, "output", "o", "raw", "Output format if other than plaintext. Allowed values are: yaml, json")
```

Comment:
This "raw" is different from the other "raw" printer in your other PR right? That one dumped the go struct (or whatever was passed) right? I'm started to see what you were doing now, sorry for the babbling stream of conscience reviews. 😊 

Raw kind of implies a dump to me (as you saw from my first fumble on your other PR). Maybe we need a different term. Either "plaintext" like you used in your description here or maybe "human" to imply that it's not really formatted any particular way.

Sorry for the walls of text on this issue. I don't think it deserves walls of text back. 😝

Reply:
Also the helptext needs a bit of clarifying. None of the options are "plaintext", so that's confusing. Also we are missing "raw" from the allowed values".

Reply:
This "raw" is indeed the same - or is intended to be the same - as the one in https://github.com/deislabs/porter/pull/133.

In implementing the `Status` action in this PR, I was looking for a way to support the default (unnamed) output of `helm status` while still utilizing porter's printer pkg for the other supported formats (json,yaml).  So, I was thinking maybe adding a "raw" -- or, perhaps better-named as "human"/"plaintext" -- printer output might prove useful elsewhere as well.

I'd say _if_ this additional output option is viable for inclusion -- let's call it `plaintext`? in https://github.com/deislabs/porter/pull/133, then I would update this logic accordingly.  

Otherwise, an alternative approach would be to support _only_ `json` or `yaml` outputs on this mixin's `Status` action... or, perhaps just handle the special `plaintext` case here and _not_ add it to porter's printer pkg.  Thoughts?

Reply:
Human is amusing, but perhaps confusing especially to non-native english speakers I'm guessing. How about plaintext (that's what you went with when you explained it in your helptext) and let's see how far that gets us? I vote for plumbing it all the way through porter so that we can really use it end to end (and not just in the mixin directly).

Reply:
👍 updated to `plaintext`; end-to-end output configurability tracking in https://github.com/deislabs/porter/issues/143 (thanks!)